textlogger/textlogger.go

```diff
@@ -94,19 +94,25 @@ func (l *tlogger) Error(err error, msg string, kvList ...interface{}) {
 func (l *tlogger) print(err error, s severity.Severity, msg string, kvList []interface{}) {
 	// Determine caller.
 	// +1 for this frame, +1 for Info/Error.
-	_, file, line, ok := runtime.Caller(l.callDepth + 2)
-	if !ok {
+	skip := l.callDepth + 2
+	file, line := l.config.co.unwind(skip)
```

Comment:
Do you think it is a good idea to have an if not nil check for the `l.config.co.unwind` ? I don't think anyone uses `&Config{}` to create the config. But just in case if they do and forget to call the `Backtrace` handler and set the `unwind` this might run into an issue. or do you think this is an overkill to add this check ?

Reply:
@pohly Just a minor comment. I am good otherwise.

Reply:
My initial reaction was that someone might indeed be doing that because we can't prevent it. But then I remembered that it's explicitly not allowed:

https://github.com/kubernetes/klog/blob/2d0829603f74534b4b4e09551c8d003da4558143/textlogger/options.go#L33-L34

If we were to add a nil check now for members, then we would enter a slippery slope of supporting something that wasn't meant to be supported and make the code more complex. Besides, `Config.output` also doesn't have a nil check, so someone who isn't following the documentation is already crashing.

No nil check?

Reply:
That is a valid point. I think we can avoid the nil check then